### ISSUE TITLE
Multiple 'abandoning brokered message' while prefetch = 0 and max concurrency =1

### ISSUE THREAD

**Issue by ramonsmits** [Particular/NServiceBus.AzureServiceBus#350]
Configured a core V6 endpoint with max concurrency level of 1 and configured V7 with a prefetchsize = 0 and I didn't expect to see multiple *Abandoning brokered message* when shutting down.

I don't know if this is a bug or expected behavior.

```
2016-09-29 11:00:05.3537|INFO|7|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
2016-09-29 11:00:05.3537|INFO|7|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Abandoning brokered message 400a3dca-9201-4699-a139-0555c794770d
2016-09-29 11:00:05.4308|INFO|12|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
2016-09-29 11:00:05.4308|INFO|12|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Abandoning brokered message 400a3dca-9201-4699-a139-0555c794770d
2016-09-29 11:00:05.4879|INFO|7|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
2016-09-29 11:00:05.4879|INFO|7|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Abandoning brokered message 400a3dca-9201-4699-a139-0555c794770d
2016-09-29 11:00:05.5446|INFO|8|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
2016-09-29 11:00:05.5446|INFO|8|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Abandoning brokered message 400a3dca-9201-4699-a139-0555c794770d
2016-09-29 11:00:05.6107|INFO|12|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
2016-09-29 11:00:05.6107|INFO|12|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Abandoning brokered message 400a3dca-9201-4699-a139-0555c794770d
2016-09-29 11:00:05.6883|INFO|7|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
2016-09-29 11:00:05.6883|INFO|7|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Abandoning brokered message 400a3dca-9201-4699-a139-0555c794770d
2016-09-29 11:00:05.7830|INFO|12|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
2016-09-29 11:00:05.7830|INFO|12|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Abandoning brokered message 400a3dca-9201-4699-a139-0555c794770d
2016-09-29 11:00:05.8404|INFO|8|NServiceBus.Transport.AzureServiceBus.MessageReceiverNotifier|Received message while shutting down, abandoning it so we can process it later.
```

**Comment by SeanFeldman** [comment]
@ramonsmits is this still happening?

**Comment by ramonsmits** [comment]
@SeanFeldman Can't seem to reproduce this anymore. Seems to be resolved by 7.0.1
